### ISSUE TITLE
Names upside down

### ISSUE THREAD

**Issue by Esmolante** [organicmaps/organicmaps#2988]
Just a small detail that I think has not been reported. In navigation mode, sometimes the name of the road on which I am driving changes reading direction very fast. It's not serious, just a little annoying.

**Comment by Esmolante** [comment]
That's right, if the road goes from North to South, the name of the road quickly alternates between reading N to S, and S to N.
Today I tried to reproduce it with no result. I saw it on the A-6 highway in Spain, between Adanero and Benavente, heading North. If it does it again, I'll record it and send it to you.

**Comment by endim8** [comment]
It's because it's trying to keep the text upright, however maybe it could have a slight buffer between 87° and 93° to avoid this jittering 

https://user-images.githubusercontent.com/26939824/180660893-6e232e47-195e-4264-9d02-485857cd9d77.mp4
